sys/include/net/gnrc/rpl.h

```diff
@@ -310,10 +318,18 @@ static inline bool GNRC_RPL_COUNTER_GREATER_THAN(uint8_t A, uint8_t B)
  * @name Parameters used for DAO handling
  * @{
  */
+#ifndef GNRC_RPL_DAO_SEND_RETRIES
 #define GNRC_RPL_DAO_SEND_RETRIES (4)
+#endif
+#ifndef GNRC_RPL_DEFAULT_WAIT_FOR_DAO_ACK
 #define GNRC_RPL_DEFAULT_WAIT_FOR_DAO_ACK (3)
+#endif
+#ifndef GNRC_RPL_REGULAR_DAO_INTERVAL
 #define GNRC_RPL_REGULAR_DAO_INTERVAL (60)
-#define GNRC_RPL_DEFAULT_DAO_DELAY (5)
+#endif
+#ifndef GNRC_RPL_DEFAULT_DAO_DELAY
+#define GNRC_RPL_DEFAULT_DAO_DELAY (1)
```

Comment:
can you put a `@see` pointing to the respective location in the RFC for this value?